geps/gep-1867.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: gateway.networking.k8s.io/v1alpha1
+kind: GEPDetails
+number: 1867
+name: Per-Gateway Infrastructure
+type: API
+status: Experimental
+authors:
+  - howardjohn
+relationships:
+  updates:
```

Comment:
Somehow "updates" doesn't feel quite right here. It's like this GEP is building on top of 1762. Maybe `dependsOn` or `extends` or some other word would fit better here?

Reply:
Sure, `extends` is probably better.